src/pyhf/optimize/opt_minuit.py

```diff
@@ -58,17 +58,16 @@ def _get_minimizer(
             wrapped_objective = objective_and_grad
             jac = None
 
-        kwargs = dict(
-            fcn=wrapped_objective,
-            grad=jac,
-            start=init_pars,
-            error=step_sizes,
-            limit=init_bounds,
-            fix=fixed_bools,
-            print_level=self.verbose,
-            errordef=self.errordef,
-        )
-        return iminuit.Minuit.from_array_func(**kwargs)
+        # FAILING:
+        # * tests/test_optim.py
+        # * tests/test_validation.py
```

Comment:
These are passing on the HEAD of `iminuit` so the next patch release this will be good to go. :+1: